sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -454,7 +454,7 @@
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=min(&#10;	max(&#10;	    if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap)&#10;		then &#10;			dmnPartTimeAwardAllowableLimits.limitAwardCSGD2OrLessChildAmount&#10;		else&#10;			max(dmnPartTimeAwardAllowableLimits.limitAwardCSGD2OrLessChildAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardCSGD2OrLessChildSlope), 0) * offeringWeeks&#10;        ,&#10;        dmnPartTimeAwardAllowableLimits.limitAwardCSGD2OrLessChildAmount&#10;    ),&#10;	dmnPartTimeAwardAllowableLimits.limitAwardCSGDAmount&#10;)" target="federalAwardCSGDAmount" />
-          <zeebe:output source="=if (awardEligibilityCSGD = true)&#10;then&#10;  if (calculatedDataTotalRemainingNeed2 &#62; federalAwardCSGDAmount)&#10;  then federalAwardCSGDAmount * offeringWeeks&#10;  else calculatedDataTotalRemainingNeed2 * offeringWeeks&#10;else&#10;0" target="federalAwardNetCSGDAmount" />
+          <zeebe:output source="=if (awardEligibilityCSGD = true)&#10;then&#10;  min(calculatedDataTotalRemainingNeed2, federalAwardCSGDAmount - programYearTotalPartTimeCSGD)&#10;else&#10;0" target="federalAwardNetCSGDAmount" />
```

Comment:
May not be part of PR, but I felt the award eligibility could be done on the first variable `federalAwardCSGDAmount` to reduce code complexity/exit fast.

![image](https://github.com/bcgov/SIMS/assets/54600590/e8fa7eeb-d179-485b-bb72-058853d3832d)

Reply:
As we discussed, if needed we can have a call with the devs and have a separate ticket created to this.